app/gui/integration-test/dashboard/copy.spec.ts

```diff
@@ -159,8 +159,8 @@ test.test('duplicate', ({ page }) =>
     .mockAllAndLogin({ page })
     // Assets: [0: New Project 1]
     .newEmptyProject()
-    .waitForEditorToLoad()
-    .goToPage.drive()
+    // .waitForEditorToLoad()
+    // .goToPage.drive()
```

Comment:
maybe instead of commenting out without any description lets put here `FIXME: Uncomment once the cloud execution in browser will be re-enabled` with gh issue for tracking? Use the same in other places